test/upload-test-images.sh

```diff
@@ -22,6 +22,7 @@ function upload_test_images() {
   # to pickup .ko.yaml
   cd "$( dirname "$0")/.."
   local image_dir="test/test_images"
+  local vendor_image_dir="vendor/knative.dev/serving/test/test_images"
```

Comment:
@itsmurugappan @navidshaikh 
I was wondering how you would feel about creating a symlink of grpc-ping from vendor dir to `./test/test-images/grpc-ping`. It'd make test dependencies appear a bit more unified. In addition there'd be a benefit for our internal repo handling. But it's just an idea how to make our life easier elsewhere. :)